uicomponent-compose/main/src/main/java/io/github/droidkaigi/feeder/viewmodel/RealFeedViewModel.kt

```diff
@@ -148,9 +149,17 @@ class RealFeedViewModel @Inject constructor(
                     playingPodcastState.value = newState
                 }
                 is FeedViewModel.Event.ReloadContent -> {
-                    repository.refresh()
+                    refreshFeed()
                 }
             }
         }
     }
+
+    private suspend fun refreshFeed() {
```

Comment:
Thank you for sharing it with the function!👍  With this name, it looks like an update of the display, so the name refleshRepository may not be misunderstood.  There is no need to fix it in this PR.